x/blob/client/cli/payforblob.go

```diff
@@ -52,27 +54,28 @@ func CmdPayForBlob() *cobra.Command {
 			"\t--keyring-backend test \\\n" +
 			"\t--fees 21000utia \\\n" +
 			"\t--yes \n",
-		Short: "Pay for a data blob(s) to be published to Celestia.",
-		Long: "Pay for a data blob(s) to be published to Celestia.\n" +
-			"User can use namespaceID and blob as argument for single blob submission \n" +
-			"or use --input-file flag with the path to a json file for multiple blobs submission, \n" +
-			`where the json file contains:
-
+		Short: "Pay for data blob(s) to be published to Celestia.",
+		Long: `Pay for data blob(s) to be published to Celestia.
+To publish a single blob, specify the namespaceID and blob via CLI arguments.
+To publish multiple blobs, use the --input-file flag with the path to a JSON file.
+The JSON should look like:
+
+{
+	"Blobs": [
 		{
-			"Blobs": [
-				{
-					"namespaceID": "0x00010203040506070809",
-					"blob": "0x48656c6c6f2c20576f726c6421"
-				},
-				{
-					"namespaceID": "0x00010203040506070809",
-					"blob": "0x48656c6c6f2c20576f726c6421"
-				}
-			]
+			"namespaceID": "0x00010203040506070809",
+			"blob": "0x48656c6c6f2c20576f726c6421"
+		},
+		{
+			"namespaceID": "0x00010203040506070809",
+			"blob": "0x48656c6c6f2c20576f726c6421"
 		}
+	]
+}
 
-		namespaceID is the user-specifiable portion of a version 0 namespace. It must be a hex encoded string of 10 bytes.\n
-		blob must be a hex encoded string of any length.\n
+The namespaceID is the user-specifiable portion of a version 0 namespace.
+The namespaceID must be a hex encoded string of 10 bytes.
+The blob must be a hex encoded string of any length.
```

Comment:
[Question] Is zero-length blob an acceptable input? if not, then it is worth mentioning that blobs should have non-zero length.

Reply:
Agreed, good call!

Reply:
2c37b373a4152e2d67faeb8d0725dbdc8e1a4444